app/forms/instance-create.tsx

```diff
@@ -468,6 +432,56 @@ export function CreateInstanceForm() {
   )
 }
 
+const AdvancedAccordion = ({
+  control,
+  isSubmitting,
+}: {
+  control: Control<InstanceCreateInput>
+  isSubmitting: boolean
+}) => {
+  const [value, setValue] = useState<string[]>([])
```

Comment:
does it still need this advancedness to solve the original problem?

Reply:
Advanced is the name of the accordion. This was just a general refactor to make it easier and neater to add to this section in the future.

Reply:
I'm talking about the `useState`, which wasn't there before.

Reply:
My bad, you're right I can do this all with css

Reply:
...or can I. We need `isOpen` to do the auto scrolling when you open an accordion I believe.